crowd_sim/envs/crowd_sim.py

```diff
@@ -149,8 +190,14 @@ def generate_random_human_position(self, human_num, rule):
                     else:
                         human = self.generate_square_crossing_human()
                     self.humans.append(human)
+        elif rule == 'static': ## Only test for generating static obstacles 
+            self.generate_random_obstacles(human_num)
+            self.humans = []
+            for i in range(human_num):
+                self.humans.append(self.generate_square_crossing_human())
```

Comment:
To support a previous comment: In cases like this, we might forget that we call `generate_random_obstacles()` in `generate_random_human_position()` and end up calling `generate_random_obstacles()` twice.

Reply:
ofcourse we do not make this mistake when we call `generate_random_obstacles` and this code is bug free, but a little error-prone.

Reply:
thanks for the reminder!